shell.nix

```diff
@@ -4,19 +4,16 @@
 , rev ? "in-nix-shell"
 , sourcesOverride ? { }
 , packages ? import ./nix { inherit crossSystem config sourcesOverride rev; }
-, pkgs ? packages.pkgs
-, localPkgs ? packages.pkgsLocal
 }:
 let
+  inherit (packages) pkgs pkgsLocal pkgsMusl;
```

Comment:
buffalo buffalo buffalo buffalo

Reply:
Serious suggestion: what if we used `nixpkgs` instead of `pkgs` everywhere to refer to the nixpkgs package set?